src/openloco/window.cpp

```diff
@@ -105,6 +105,16 @@ namespace openloco::ui
         return (this->holdable_widgets & (1ULL << index)) != 0;
     }
 
+    // 0x0045A0B3
+    void window::drawViewports(gfx::drawpixelinfo_t* dpi)
+    {
+        if (viewports[0] != nullptr)
+            viewports[0]->render(dpi, gfx::point_t(dpi->x, dpi->y), gfx::ui_size_t(dpi->width, dpi->height));
```

Comment:
I don't really see a point for passing the `gfx::point_t`s here, as they are properties of the `dpi` anyway.